Library/Homebrew/.rubocop.yml

```diff
@@ -45,10 +45,6 @@ Naming/PredicateName:
   # Can't rename these.
   AllowedMethods: is_32_bit?, is_64_bit?
 
-Style/FrozenStringLiteralComment:
-  Exclude:
-    - '**/Casks/**/*.rb'
```

Comment:
Same as above for being more explicit about what this is targetting (the test fixtures).

Reply:
Wait I misread this, this was a deletion. 🙄 Ignore me.